src/heron.py

```diff
@@ -56,9 +56,12 @@ def create_componentsets_in_HERON(comp_sets_folder, heron_input_xml):
 
   # Goin through the FORCE componentSets to extract relevant info
   for textfile in comp_set_files_list:
-    if textfile.startswith('componentSet'):
+    if textfile.startswith('componentSet') and (textfile.endswith('.json') or textfile.endswith('.txt')):
       textfile_path = comp_sets_folder+"/"+textfile
-      comp_set_dict = json.load(open(textfile_path))
+      try:
+        comp_set_dict = json.load(open(textfile_path))
```

Comment:
it would be better to open the file using a `with` statement as mentioned here: https://pylint.readthedocs.io/en/latest/user_guide/messages/refactor/consider-using-with.html